vscode-extension/src/json-schemas/relay-config-json.jsonc

```diff
@@ -0,0 +1,588 @@
+              "description": "The maximum number concurrent requests that will be made to `url`",
+              "minimum": 1
+            }
+          }
+        },
+        {
+          "type": "object",
+          "title": "Local Persisting",
+          "required": ["file"],
+          "properties": {
+            "file": {
+              "type": "string",
+              "description": "Path to the JSON file that will contain operations map"
+            }
+          }
+        }
+      ]
+    },
+    "SchemaConfig": {
+      "type": "object",
+      "description": "Configuration where Relay should expect some fields in the schema.",
+      "properties": {
+        "connectionInterface": {
+          "type": "object",
+          "properties": {
+            "cursor": {
+              "type": "string"
+            },
+            "edges": {
+              "type": "string"
+            },
+            "endCursor": {
+              "type": "string"
+            },
+            "hasNextPage": {
+              "type": "string"
+            },
+            "hasPreviousPage": {
+              "type": "string"
+            },
+            "node": {
+              "type": "string"
+            },
+            "pageInfo": {
+              "type": "string"
+            },
+            "startCursor": {
+              "type": "string"
+            }
+          }
+        },
+        "nodeInterfaceIdField": {
+          "type": "string",
+          "description": "Configure the name of the globally unique ID field on the Node interface",
+          "default": "id"
+        },
+        "nonNodeIdFields": {
+          "type": "object",
+          "required": ["allowedIdTypes"],
+          "properties": {
+            "allowedIdTypes": {
+              "type": "object",
+              "description": "Mappings from types in your schema to allowed types for their fields named `id`",
+              "examples": [{"ObjectType": "CustomIdType"}],
+              "additionalProperties": {
+                "description": "The type of the `id` field of this type",
+                "type": "string",
+                "minLength": 1
+              }
+            }
+          }
+        },
+        "unselectableDirectiveName": {
+          "type": "string",
+          "description": "The name of the directive indicating fields that cannot be selected"
+        }
+      }
+    },
+    "DevVariableName": {
+      "type": "string",
+      "description": "Name of the global variable for dev mode",
+      "default": "__DEV__"
+    },
+    "FeatureFlags": {
+      "type": "object",
+      "properties": {
+        "enable_flight_transform": {
+          "type": "boolean"
+        },
+        "enable_relay_resolver_transform": {
+          "type": "boolean"
+        },
+        "use_named_imports_for_relay_resolvers": {
+          "type": "boolean",
+          "default": true
+        },
+        "relay_resolver_model_syntax_enabled": {
+          "type": "boolean"
+        },
```

Comment:
Should feature flags be properly typed or should I just not type them as they are likely very transient?